transforms/transforms_test.go

```diff
@@ -263,106 +264,7 @@ var transformEqualities = []tests.TableCompareConfig{
-		{"header1": "value5", "header2": "value6", "header3": "value5", "header4": "value9"},
-	}
-
-	leftTable := slice.New(defaultInput())
-	rightTable := slice.New([]optimus.Row{
-		// 'value1' in left table maps to no rows in the right table
-		{"header3": "valueNoMatch", "header4": "value7"},
-		{"header3": "value3", "header4": "value8"},
-		{"header3": "value5", "header4": "value9"},
-	})
-	combinedTable := optimus.Transform(leftTable, Join(rightTable, "header1", "header3", JoinType.Inner))
-	rows := tests.HasRows(t, combinedTable, 2)
-	assert.Equal(t, expected, rows)
-}
-
-func TestJoinOneToMany(t *testing.T) {
-	expected := []optimus.Row{
-		{"header1": "value1", "header2": "value2", "header3": "value1", "header4": "value8"},
-		{"header1": "value1", "header2": "value2", "header3": "value1", "header4": "value9"},
-	}
-
-	leftTable := slice.New([]optimus.Row{
-		{"header1": "value1", "header2": "value2"},
-	})
-	rightTable := slice.New([]optimus.Row{
-		// 'value1' in left table maps to two rows in the right table
-		{"header3": "value1", "header4": "value8"},
-		{"header3": "value1", "header4": "value9"},
-	})
-	combinedTable := optimus.Transform(leftTable, Join(rightTable, "header1", "header3", JoinType.Inner))
-	rows := tests.HasRows(t, combinedTable, 2)
-	assert.Equal(t, expected, rows)
-}
-
-func TestJoinManyToOne(t *testing.T) {
-	expected := []optimus.Row{
-		{"header1": "value1", "header2": "value2", "header3": "value1", "header4": "value8"},
-		{"header1": "value1", "header2": "value3", "header3": "value1", "header4": "value8"},
-	}
-	leftTable := slice.New([]optimus.Row{
-		{"header1": "value1", "header2": "value2"},
-		{"header1": "value1", "header2": "value3"},
-	})
-	rightTable := slice.New([]optimus.Row{
-		{"header3": "value1", "header4": "value8"},
-	})
-
-	combinedTable := optimus.Transform(leftTable, Join(rightTable, "header1", "header3", JoinType.Inner))
-
-	rows := tests.HasRows(t, combinedTable, 2)
-	assert.Equal(t, expected, rows)
-}
-
-func TestJoinManyToMany(t *testing.T) {
-	expected := []optimus.Row{
-		{"header1": "value1", "header2": "value2", "header3": "value1", "header4": "value4"},
-		{"header1": "value1", "header2": "value2", "header3": "value1", "header4": "value5"},
-		{"header1": "value1", "header2": "value3", "header3": "value1", "header4": "value4"},
-		{"header1": "value1", "header2": "value3", "header3": "value1", "header4": "value5"},
-	}
-	leftTable := slice.New([]optimus.Row{
-		{"header1": "value1", "header2": "value2"},
-		{"header1": "value1", "header2": "value3"},
-	})
-	rightTable := slice.New([]optimus.Row{
-		{"header3": "value1", "header4": "value4"},
-		{"header3": "value1", "header4": "value5"},
-	})
-
-	combinedTable := optimus.Transform(leftTable, Join(rightTable, "header1", "header3", JoinType.Inner))
-
-	rows := tests.HasRows(t, combinedTable, 4)
-	assert.Equal(t, expected, rows)
-}
-
-func TestLeftOverwritesRight(t *testing.T) {
+func TestJoinMergePairs(t *testing.T) {
```

Comment:
I agree with removing most of the tests, I was just wanting to make sure we had some part of the code that tested the LeftJoin case in the merge logic since it is slightly different.